### PR TITLE
Use microscope CI image in ginkgo tests

### PR DESCRIPTION
**Pull request by nebril**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4898)

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

File: test/k8sT/manifests/microscope.yaml [hunk 1]
```diff
@@ -69,6 +69,6 @@ spec:
   - args:
     - sleep
     - "100000"
-    image: docker.io/cilium/microscope:1.0.1
+    image: docker.io/cilium/microscope:ci
```
Comment (aanm) [inline test/k8sT/manifests/microscope.yaml]: Can't we have a version? something like `ci-x.y.z`? or `x.y.z-ci`

**Review by aanm — CHANGES_REQUESTED** [review]

File: test/k8sT/manifests/microscope.yaml (hunk 1 above)
Comment (nebril) [reply test/k8sT/manifests/microscope.yaml]: we can, will do

**Comment by ianvernon** [comment]
@nebril can you elaborate on how this is fixing #4879 ? I don't understand what the issue was in #4879 that necessitates a change in the image. In other words, what benefits does using this image provide in the CI vs. the existing microscope image? Please update the commit / PR description accordingly.

**Comment by nebril** [comment]
@ianvernon it seems like I misunderstood #4879, but we still need to change this manifest in order to provide timed logs to ci output, so I will fix this pr per Andres suggestion.

File: test/k8sT/manifests/microscope.yaml (hunk 1 above)
Comment (nebril) [reply test/k8sT/manifests/microscope.yaml]: done

**Comment by nebril** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Comment by nebril** [comment]
@ianvernon I was wrong about being wrong xD. Seems like microscope failed because pgrep was not installed in base microscope image.

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
Seems like root issue was caused by python 3.7 being released lately and change of importlib API. Pinned version to 3.6.6 in microscope repo

**Pull request merged**